docusaurus/docs/user-docs/getting-started/setting-up-admin-panel.md

```diff
@@ -0,0 +1,110 @@
+| -------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| First name           | Write your first name in the textbox.                                                                                                                                                                             |
+| Last name            | Write your last name in the textbox.                                                                                                                                                                              |
+| Email                | Write your complete email address in the textbox.                                                                                                                                                                 |
+| Username             | (optional) Write a username in the textbox.                                                                                                                                                                       |
+| Interface language   | Among the drop-down list, choose a language for your Strapi application interface.                                                                                                                                |
+| Interface mode       | Among the drop-down list, choose a mode for your Strapi application interface: either "Light mode" or "Dark mode". Note that by default, the chosen mode for a Strapi application is based on the browser's mode. |
+
+
+4. Click on the **Save** button.
+
+
+### Changing your password
+
+
+To change the password of your account:
+
+
+1. Go to your administrator profile.
+2. Fill in the password-related options:
+
+
+| Password modification |                                                                                                                  |
+| --------------------- | ---------------------------------------------------------------------------------------------------------------- |
+| Current password      | Write your current password in the textbox. <br/> 💡 You can click on the eye icon for the password to be shown. |
+| Password              | Write the new password in the textbox. <br/> 💡 You can click on the eye icon for the password to be shown.      |
+| Password confirmation | Write the same new password in the textbox. <br/> 💡 You can click on the eye icon for the password to be shown. |
+
+
+3. Click on the **Save** button.
+
+
+---
+
+
+Congratulations on being a new Strapi user! You're now ready to discover all the features and options that Strapi has to offer!
+
+
+Head back to the [Welcome Page](../intro.md).
```

Comment:
This sentence l.110 looks very weird/out of place in my opinion 🤔

Reply:
I included a link back to the welcome page on that line. I've taken it out now.